app/lib/kafka.server.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Kafka } from 'gcn-kafka'
+
+import { domain, getEnvOrDie, hostname } from './env.server'
+
+const client_id = getEnvOrDie('KAFKA_CLIENT_ID')
+const client_secret = getEnvOrDie('KAFKA_CLIENT_SECRET')
+const producer = new Kafka({
+  client_id,
+  client_secret,
+  domain,
+}).producer()
+
+process.on('SIGTERM', async () => {
+  await producer.disconnect()
+  process.exit(0)
+})
+
+export async function sendKafka(item: any, topic: string) {
```

Comment:
Switch order of arguments to more closely match the arguments to producer.send.

```suggestion
export async function sendKafka(topic: string, value: string) {
```